src/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+    <title>Botfuel WebChat Demo</title>
+</head>
+
+<body>
+    <div id="botfuel"></div>
+</body>
+<script>
+    BotfuelWebChat.init({
+        applicationId: '1409615561904',
+        theme: {
+            colors: {
+                main: '#3d3d3d',
+                mainLight: '#f5f5f5',
+                primary: '#f4b600',
```

Comment:
we'll need to change these before the end of the sprint, I guess it won't look good on the page